src/Microsoft.ML.CodeGenerator/Templates/Console/ModelBuilder.cs

```diff
@@ -116,8 +120,8 @@ public static void CreateModel()
                                            } 
             this.Write(";\r\n");
 }
-            this.Write("\r\n            // Set the training algorithm \r\n            var trainer = mlContext" +
-                    ".");
+ if(Trainer != String.Empty ) {
```

Comment:
spacing nit: (yes, we'd have to find it in the corresponding .tt file)
```suggestion
 if (Trainer != String.Empty) {
```

Also, it's likely that it should be further indented